### ISSUE TITLE
Jest 28.0.0 and jest-environment-jsdom fails to import use-debounce in tests

### ISSUE THREAD

**Issue by Glup3** [xnimorz/use-debounce#132]
**Describe the bug**
Importing the package does not work since the Jest 28.0.0 update in combination with `jest-environment-jsdom`.

**To Reproduce**

1. update jest to 28.0.0
2. use jest-environment-jsdom (28.0.0)
3. create a test that uses use-debounce
4. run the test

**Expected behavior**
The test should run successfully.

**use-debounce version:** 8.0.0

**Additional Information**
This problem is also in other packages (described in their docs) https://jestjs.io/docs/upgrading-to-jest28#packagejson-exports
Here is the analysis of the problem and their workaround, which works:
https://github.com/microsoft/accessibility-insights-web/pull/5421#issuecomment-1109168149
https://github.com/microsoft/accessibility-insights-web/pull/5421/commits/9ad4e618019298d82732d49d00aafb846fb6bac7

**Comment by sibelius** [comment]
this is breaking our server side rendering with webpack as well

**Comment by sibelius** [comment]
removing

```jsx
"exports": {
    ".": {
      "browser": "./dist/index.module.js",
      "umd": "./dist/index.umd.js",
      "import": "./dist/index.mjs",
      "require": "./dist/index.js",
      "default": "./dist/index.module.js"
    }
  },
```

from package.json solved the problem

**Comment by xnimorz** [comment]
Hey @Glup3 and @sibelius !

Sorry, for the long response, and huge thanks to both of you!

I updated the exports section https://github.com/xnimorz/use-debounce/commit/dd377cbbc8b75bf4c9da477a969be50584d23e79 and tested on the local project with `jest@28` and `jest-environment-jsdom`, so the new version should work properly. 

I've published `use-debounce@8.0.1`, you can update to this version. 

Let me know if everything is okay for you

**Comment by xnimorz** [comment]
Great, thanks for the info
